src/resource_clients/actor.ts

```diff
@@ -56,8 +57,8 @@ export class ActorClient extends ResourceClient {
      * Starts an actor and immediately returns the Run object.
      * https://docs.apify.com/api/v2#/reference/actors/run-collection/run-actor
      */
-    async start(input?: unknown, options: ActorStartOptions = {}): Promise<ActorRun> {
-        // input can be anything, pointless to validate
+    async start(input?: string | Dictionary, options: ActorStartOptions = {}): Promise<ActorRun> {
+        ow(input, ow.optional.any(ow.object, ow.string));
```

Comment:
Actor input can be anything. This is a breaking change imo.

Reply:
The API begs to differ, and rejects anything that isn't an object, so shouldn't we also validate for that?

Reply:
> btw the type accepts also `undefined`, which would fail this runtime validation

It wouldn't fail the runtime validation, since the entire predicate is optional (tested this too)

Reply:
The API will accept whatever you set the `content-type` to. E.g. if you set it to `application/pdf` then it will process it as a buffer.

This does not apply to tasks. Tasks accept only JSON input.

Reply:
Ah, its a case of Actor runs vs Tasks.. Will revert tomorrow 👍